client/src/components/PlotContent.js

```diff
@@ -1,57 +1,81 @@
 import React, { useState } from "react";
 import { Button } from "@mui/material";
 import TextField from '@mui/material/TextField';
+import Plot from 'react-plotly.js';
 import Dialog from '@mui/material/Dialog';
 import DialogActions from '@mui/material/DialogActions';
 import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 import DialogTitle from '@mui/material/DialogTitle';
 import { ButtonGroup } from "@mui/material";
 import { teal } from "@mui/material/colors";
+import EditIcon from '@mui/icons-material/Edit';
 
 const primary = teal[500];
 
-function PlotContent(props){
+function PlotContent(props) {
 
     const [open, setOpen] = React.useState(false);
     const [selectedBtn, setSelectedBtn] = React.useState(1);
 
     const activeColor = teal[500]
 
     const handleClickOpen = () => {
-      setOpen(true);
+        setOpen(true);
     };
-  
+
     const handleClose = () => {
-      setOpen(false);
+        setOpen(false);
     };
 
     return (
-        <div>
-            <p>
-                <Button variant="outlined" onClick={handleClickOpen}>
-                    {props.plotName}
+        <div class="plotContainer">
+            <Plot
+                data={[
+                    {
+                        x: [1, 2, 3, 4, 5, 6, 7],
+                        y: [2, 6, 3, 5, 1, 8, 3],
+                        type: 'scatter',
+                        mode: 'lines',
+                        marker: { color: 'red' },
+                    }
+                ]}
+                layout={{
+                    height: 300,
+                    className: "plotGraph",
+                    margin: {
+                        l: 50,
+                        r: 30,
+                        b: 50,
+                        t: 30,
+                        pad: 15
+                      }
+                }}
+                config={{responsive: true}}
```

Comment:
Does this work? Since you stated the plotly graphs are in fact not reponsive yet